base/lib/core/presenter/AboutDialog.cpp

```diff
@@ -0,0 +1,182 @@
+  softwareList->setHorizontalScrollBarPolicy(Qt::ScrollBarAlwaysOff);
+  softwareList->setVerticalScrollBarPolicy(Qt::ScrollBarAlwaysOff);
+
+  softwareList->setStyleSheet(R"_(
+                              QListView::item:!selected:hover{
+                              background-color:#415491;
+                              color: #ffffff;
+                              }
+                              QListView::item:selected:active, QListView::item:selected:!active{
+                              background-color:#73C1C6;
+                              color: #1A2024;
+                              }
+                              QListView {
+                              background-color: rgb(18,23,26);
+                              margin:0px;
+                              }
+                              /* QListView::item{
+                              color: rgb(0,146,207);
+                              }*/
+                              QScrollBar::right-arrow:horizontal, QScrollBar::left-arrow:horizontal
+                              {
+                              border: none;
+                              background: none;
+                              color: none;
+                              }
+                              QScrollBar::add-line:vertical {
+                              border: none;
+                              background: none;
+                              }
+                              QScrollBar::sub-line:vertical {
+                              border: none;
+                              background: none;
+                              }
+                              )_");
+  for(auto item : map)
+  {
+    softwareList->addItem(item.first);
+  }
+
+
+  // license
+  auto license = new QPlainTextEdit{this};
+  license->move(280,230);
+  license->resize(185,183);
+  license->setFont(m_catamaranFont);
+  license->setReadOnly(true);
+  connect(softwareList,&QListWidget::currentTextChanged,this,
+          [=] (const QString& currentText)
+  {
+    license->setPlainText(map.at(currentText));
+  });
+}
+
+void AboutDialog::mousePressEvent(QMouseEvent *event)
+{
+
+  QPointF pos = event->localPos();
+  if(m_mouseAreaLabri.contains(pos))
+  {
+    QDesktopServices::openUrl(QUrl("https://www.labri.fr/"));
+  }
+  else if(m_mouseAreaBlueYeti.contains(pos))
+  {
+    QDesktopServices::openUrl(QUrl("http://www.blueyeti.fr/"));
+  }
+  else if(m_mouseAreaScrime.contains(pos))
+  {
+    QDesktopServices::openUrl(QUrl("https://scrime.u-bordeaux.fr/"));
+  }
+  this->close();
+}
+
+void AboutDialog::mouseMoveEvent(QMouseEvent *event)
+{
+  QPointF pos = event->localPos();
+  if( m_mouseAreaLabri.contains(pos)
+      || m_mouseAreaBlueYeti.contains(pos)
+      || m_mouseAreaScrime.contains(pos))
+  {
+    this->setCursor(Qt::PointingHandCursor);
+  }
+  else
+  {
+    this->setCursor(Qt::ArrowCursor);
+  }
+}
+void AboutDialog::paintEvent(QPaintEvent *event)
+{
+  QPen textPen(QColor("#0092cf"));
```

Comment:
hmmm... if these colors are reused somewhere else maybe they could go in some palette that would also be used for the splash screen for instance ? (not for today)